### PR TITLE
Docker profile added

### PR DESCRIPTION
**Pull request by thetuxracer**
I took the liberty of going ahead and submitting a PR, since a docker profile was supposed to be added. Please go ahead and critique or suggest changes, I'm pretty sure I've messed up somewhere.

### CODE REVIEW

**Comment by iankko** [comment]
@openscap-jenkins 
[add to whitelist]

File: RHEL/7/input/xccdf/services/docker.xml [hunk 1]
```diff
@@ -0,0 +1,90 @@
+<Group id="docker">
+<title>Docker Daemons</title>
+<description>The cron and at services are used to allow commands to
+be executed at a later time. The cron service is required by almost
+all systems to perform necessary maintenance tasks, while at may or
+may not be required on a given system. Both daemons should be
+configured defensively.</description>
+
+<Rule id="service_crond_enabled" severity="medium">
```
Comment (iankko) [inline RHEL/7/input/xccdf/services/docker.xml]: There already exists ```service_crond_enabled``` rule in ```RHEL/7/input/xccdf/services/cron.xml```. What this means being if the Docker profile should utilized the ```service_crond_enabled``` rule, it's enough to reference that rule in the Docker profile already.

File: RHEL/7/input/xccdf/services/docker.xml [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+<rationale>Due to its usage for maintenance and security-supporting tasks,
+enabling the cron daemon is essential.
+</rationale>
+<ident cce="27323-5" />
+<oval id="service_crond_enabled" />
+<ref nist="CM-7" cis="6.1.2" />
+<tested by="DS" on="20121024"/>
+</Rule>
+
+<Rule id="disable_anacron">
```
Comment (iankko) [inline RHEL/7/input/xccdf/services/docker.xml]: Same for ```disable_anacron``` rule.

File: RHEL/7/input/xccdf/services/docker.xml [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+that <tt>cron</tt> jobs are scheduled to run. On systems which do not require this
+additional functionality, <tt>anacron</tt> could needlessly increase the possible
+attack surface for an intruder.
+</rationale>
+<ref nist="CM-7" />
+<ident cce="80344-5" />
+</Rule>
+
+
+<Rule id="service_atd_disabled">
```
Comment (iankko) [inline RHEL/7/input/xccdf/services/docker.xml]: Ditto.

**Comment by thetuxracer** [comment]
Closing PR because its basically incorrect. I was supposed to submit a profile, but ended up creating a  XCCDF item for the report.

**Pull request closed without merging**